src/client/content_cao.cpp

```diff
@@ -563,6 +566,23 @@ void GenericCAO::removeFromScene(bool permanent)
 	}
 }
 
+void GenericCAO::setSceneNodeMaterial(video::E_MATERIAL_TYPE material_type)
+{
+	scene::ISceneNode *node = getSceneNode();
+
+	node->setMaterialFlag(video::EMF_LIGHTING, false);
+	node->setMaterialFlag(video::EMF_BILINEAR_FILTER, false);
+	node->setMaterialFlag(video::EMF_FOG_ENABLE, true);
+
+	if (m_enable_shaders) {
+		node->setMaterialType(m_material_type);
```

Comment:
what is `m_material_type` and why is it different (is it?) from `material_type`?

Reply:
I renamed parameter names, so now intent should be clear even w/o comments